LICENSE.md

```diff
@@ -672,6 +672,28 @@ https://github.com/KhronosGroup/glTF-WebGL-PBR
 >CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
 >OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+### deck.gl
```

Comment:
Add a link to the repo and use quoted text for the license text, similar to the others.